test/edge-cases.html

```diff
@@ -29,11 +29,54 @@
 				$('#powertip-css').attr('href', '../css/jquery.powertip' + theme + '.css');
 			}
 
+			// css position switcher allows testing html and body CSS position values that
+			// changes the origin from which the tooltip is positioned.
+			function setCssPosition() {
+				var attributeSets = {
+					// default positioning is static
+					static:   { position: '',         left: '',     right: '',      top: '',     bottom: '' },
+					absolute: { position: 'absolute', left: '50px', right: '100px', top: '25px', bottom: '75px' },
+					relative: { position: 'relative', left: '50px', right: '100px', top: '25px', bottom: '75px' },
+					fixed:    { position: 'fixed',    left: '50px', right: '100px', top: '25px', bottom: '75px' },
+				};
+				var posType = $('#position-switcher').val();
+				var $html = $(document.documentElement);
+				var $body = $(document.body);
+				$html.css(attributeSets['static']);
+				$body.css(attributeSets['static']);
+				switch(posType) {
+					case 'html-relative':
+						$html.css(attributeSets['relative']);
+						break;
+					case 'html-fixed':
+						$html.css(attributeSets['fixed']);
+						break;
+					case 'html-absolute':
+						$html.css(attributeSets['absolute']);
+						break;
+					case 'body-relative':
+						$body.css(attributeSets['relative']);
+						break;
+					case 'body-fixed':
+						$body.css(attributeSets['fixed']);
+						break;
+					case 'body-absolute':
+						$body.css(attributeSets['absolute']);
+						break;
+					default:
+						// attributes clear above
+						break;
+				}
+				// Trigger resize to force recalculation of position compensation
+				window.dispatchEvent(new Event('resize'));
+			}
+
 			// run theme switcher on page load
 			setTheme();
 
-			// hook theme switcher to select change
+			// hook theme and position switcher to select change
 			$('#theme-switcher').on('change', setTheme);
+			$('#position-switcher').on('change', setCssPosition);
```

Comment:
I would think it will need more than the function call.  The dropdown value would presumably reset on reload unless the value is saved somewhere.  I could use a url parameter to preserve the position setting so it can be restored on reload.  Would that be helpful?